pkg/collector/corechecks/system/file_handles_freebsd_test.go

```diff
@@ -0,0 +1,42 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+// +build freebsd
+
+package system
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/DataDog/datadog-agent/pkg/aggregator/mocksender"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/blabber/go-freebsd-sysctl/sysctl"
+
+	"bou.ke/monkey"
```

Comment:
After double-checking this dep's license, I don't think we can use it: https://github.com/bouk/monkey/blob/master/LICENSE.md

To avoid adding this dep to monkey patch the function in the test, as a workaround, can you define `sysctl.GetInt64` as a variable in `linux_handle_freebsd.go`, similar to what's done on the [cpu check](
https://github.com/DataDog/datadog-agent/blob/e7f7afa6bc32339fb2ca5b6fb5afc19b1e6165e2/pkg/collector/corechecks/system/cpu.go#L24-L25)? A little hacky but does the job here.

Reply:
I'll do my best since I'm very much beginner in golang.